Documentation/ApiOverview/PasswordPolicies/_PrivateKeyGeneratorCommand.php

```diff
@@ -18,7 +18,11 @@
 )]
 final class PrivateKeyGeneratorCommand extends Command
 {
-    // inject $myService
+    // Implement class MyService
+    public function __construct(private readonly MyService $myService)
+    {
+    }
```

Comment:
```suggestion
        parent::__construct()
    }
```

The parent constructor call is mandatory for commands.

Reply:
Ehm, it is there, see above?

Reply:
Argh, GitHub displayed only a part of the change .... sorry for the noise.

Reply:
you caught me inbetween two commits that were like 2 min away ;)